registry/batch/views.py

```diff
@@ -16,9 +16,20 @@
 
 
 @blueprint.get("/import/")
+@blueprint.get("/import/<rodne_cislo>")
 @login_required
-def import_data():
+def import_data(rodne_cislo=None):
     import_form = ImportForm()
+    if rodne_cislo:
+        import_form.donation_center_id.default = -1
+        import_form.process()
+        last_record = (
+            Record.query.join(Batch)
+            .filter(Record.rodne_cislo == rodne_cislo)
+            .order_by(Batch.imported_at.desc())
+            .first_or_404()
+        )
+        import_form.input_data.data = last_record.as_original(donation_count="<POČET>")
```

Comment:
Naprostá drobnost: tady to `<POČET>` je určitě dobré v tom, že ukazuje, kde je potřeba doplnit hodnotu. Kdybych ale chtěla dvojklikem zvýraznit to, co je potřeba nahradit, tak mi tam zbudou ty „závorky“ a stejně je musím mazat. Případně, když to vyplním, tak dostanu nevallidní import. Zda to třeba nenahradit něčím jiným?

Reply:
Obávám se, že žádný znak, kterým by se to slovo dalo zvýraznit, se neoznačí společně s textem, ale klidně to může být i bez zvýraznění, protože když to nevyplní, import stejně neprojde a chyba pak dává celkem smysl, takže uživatele navede. Smažu tedy závorky.

Reply:
> Obávám se, že žádný znak, kterým by se to slovo dalo zvýraznit, se neoznačí společně s textem, ale klidně to může být i bez zvýraznění, protože když to nevyplní, import stejně neprojde a chyba pak dává celkem smysl, takže uživatele navede. Smažu tedy závorky.

Když se tam dají podtržítka, tak se to označí i s nimi, ale zase nevím, o jak moc jsou pozdržítka výraznější než to slovo samotné.

Reply:
Podtržítka jsou skvělý nápad. Dal jsem je tam.